sky/engine/core/painting/PictureRecorder.cpp

```diff
@@ -34,21 +34,12 @@ PassRefPtr<Picture> PictureRecorder::endRecording()
     RefPtr<Picture> picture = Picture::create(
         adoptRef(m_pictureRecorder.endRecording()));
     m_canvas->clearSkCanvas();
+    m_canvas->ClearDartWrapper();
     m_canvas = nullptr;
+    ClearDartWrapper();
     return picture.release();
 }
 
-PassRefPtr<Drawable> PictureRecorder::endRecordingAsDrawable()
```

Comment:
That's possible. I am concerned at the complexity of using the layer system though. Some of our lowest-level demos doubled in size when we added scenes.